### ISSUE TITLE
💅 CSS font-feature-values rules are not ignored by noUnknownProperty

### ISSUE THREAD

**Issue by jacobrask** [biomejs/biome#6567]
### Environment information

```bash
Biome 5f42630
```

### Rule name

lint/correctness/noUnknownProperty

### Playground link

https://biomejs.dev/playground/?code=QABmAG8AbgB0AC0AZgBlAGEAdAB1AHIAZQAtAHYAYQBsAHUAZQBzACAAIgBUAGUAcwB0ACIAIAB7AAoAIAAgAEAAcwB0AHkAbABlAHMAZQB0ACAAewAKACAAIAAgACAAcwBvAG0AZQAtAGYAZQBhAHQAdQByAGUAOgAgADEAOwAKACAAIAB9AAoAfQAKAA%3D%3D&language=css

### Expected result

Any declarations are allowed inside the @namespace rules in @font-feature-values. It's how you define custom idents. The `noUnknownProperty` rule should not trigger.

See https://developer.mozilla.org/en-US/docs/Web/CSS/@font-feature-values#using_styleset_in_a_font-feature-values_rule

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
